tests/jballerina-unit-test/src/test/resources/test-src/annotations/annot_access.bal

```diff
@@ -429,6 +431,135 @@ function testListExprInConstAnnot() {
     assertEquality(["key1", "key2"], config.key);
 }
 
+annotation record {| string name; |} ObjectAnnot on type;
```

Comment:
```suggestion
annotation record {|string name;|} ObjectAnnot on type;
```

Reply:
I find the current version easier to read.

Reply:
The issue in there is Ballerina formatter will format this to the `annotation record {|string name;|} ObjectAnnot on type`

Reply:
I would expect this to be something configurable eventually, given that both ways are used. There are a few other places where we deviate from what the formatter does by default at the moment too.